README.md

```diff
@@ -9,6 +10,11 @@
 4. Run `gw dockerComposeUp` to enable proxy
 5. Open `nnbuilder.ipr` file in Intellij (on macOS run `open *.ipr`)
 
+## Install database on MAC
+ * brew install postgres
+ * pg_ctl -D /usr/local/var/postgres start
+ * psql postgres
```

Comment:
Why do you need to run the cli?

Reply:
this is just a tutorial to make a DB in localhost...